src/meshapi/tests/test_admin_change_view.py

```diff
@@ -464,9 +465,11 @@ def test_add_new_planned_node_no_nn(self):
         self.assertEqual(node.notes, "Test notes")
 
     def test_add_building(self):
+        enable_flag("EDIT_PANORAMAS")
```

Comment:
It might make more sense to change these tests to avoid editing the panos, since this is the state prod will be in, right?

Reply:
Good call, added a test to add a building with the panorama editor disabled 👍🏻